src/selection_tools.jl

```diff
@@ -311,9 +318,9 @@ julia> dst=select_region(a,new_size=(10,10), dst_center=(1,1)) # pad a with zero
  0.0  0.0  0.0  0.0  0.0  0.0  2.0  2.0  2.0  2.0
 ```
 """
-function select_region!(src, dst; new_size=size(dst), 
-                        center=size(src).÷2 .+1, dst_center=size(dst).÷ 2 .+1, operator! =assign_to!)
-    new_size = Tuple(expand_size(new_size, size(dst)))    
+function select_region!(src::AbstractArray{T, N}, dst; new_size=ntuple((d)->typemax(Int)÷2,Val(N)), 
```

Comment:
still think that typemax is hacky.
even -1 would be better

Reply:
Do you prefer this version? I hope it is correct for even and odd sizes alike.